lib/Migration/Version01022Date20221202161257.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OCA\UserOIDC\Migration;
+
+use Closure;
+use Doctrine\DBAL\Schema\SchemaException;
+use OCP\DB\ISchemaWrapper;
+use OCP\DB\Types;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+/**
+ * Auto-generated migration step: Please modify to your needs!
+ */
+class Version01022Date20221202161257 extends SimpleMigrationStep {
+
+	/**
+	 * @param IOutput $output
+	 * @param Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 * @throws SchemaException
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options) {
+
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+
+		if ($schema->hasTable('user_oidc_sessions')) {
+			$table = $schema->getTable('user_oidc_sessions');
+			$indexes = $table->getIndexes();
+			foreach ($indexes as $index) {
+				// fix created_at index which is not unique
+				if ($index->isUnique() && $index->hasColumnAtPosition('created_at')) {
```

Comment:
Shouldn't that also check for the name of the index? to not remove and add the one fixed in the previous migration on fresh installations? ;)

Reply:
As it check if it's unique, no risk to remove the correct one potentially added with a fresh install.

Reply:
Right :)